backend/src/types.ts

```diff
@@ -964,6 +965,19 @@ export enum TolerationEffect {
   NO_EXECUTE = 'NoExecute',
 }
 
+export type Identifier = {
```

Comment:
Could we be more specific and name it `NodeResourceIdentifier` ?

Reply:
@dpanshug , the spec component resemble this
`export type HardwareProfileKind = K8sResourceCommon & {
  metadata: {
    name: string;
  };
  spec: {
    displayName: string;
    enabled: boolean;
    description?: string;
    tolerations?: Toleration[];
    identifiers?: Identifier[];
    nodeSelectors?: NodeSelector[];
  };`